src/scenes/home/footer/footer.js

```diff
@@ -39,6 +39,7 @@ class Footer extends Component {
               {/* <Link to="involved">Get Involved</Link>
               <Link to="blog">Blog</Link> */}
               <Link to="contact">Contact</Link>
+              <Link to="privacy">Privacy</Link>
```

Comment:
You can simply change this to 
`<a href="https://www.iubenda.com/privacy-policy/8174861" target="_blank" rel="noopener noreferrer">Privacy</a>`

and remove the changes to /home and /privacy policy.

The work you did was perfect for if we were adding a new page, but I believe we're just linking externally to a policy page for now. Thank you so much for this work - I didn't understand how it was to be implemented, until you made the PR.